recipes/keystone/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.1)
```

```suggestion
cmake_minimum_required(VERSION 3.15)
```

Conan 2.x requires CMake >=3.15: https://github.com/conan-io/tribe/blob/main/design/004-tools-cmake.md